.github/workflows/asana.yml

```diff
@@ -0,0 +1,14 @@
+name: Asana integration
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  asana:
+    uses: mbta/workflows/.github/workflows/asana.yml@v4
+    with:
+      attach-pr: true
+      trigger-phrase: "\\*\\*Asana Ticket:\\*\\*"
```

Comment:
Same "ticket" vs. "task" issue here as in Scully.

Reply:
Thanks, fixed. Didn't realize because this worked even with the trigger phrase incorrect